components/prism-log.js

```diff
@@ -93,7 +97,7 @@ Prism.languages.log = {
 		alias: 'number'
 	},
 	'time': {
-		pattern: /\b\d{1,2}:\d{1,2}:\d{1,2}(?:[.,:]\d+)?(?:\s?[+-]\d{2,4}|Z)?\b/,
+		pattern: /\b\d{1,2}:\d{1,2}:\d{1,2}(?:[.,:]\d+)?(?:\s?[+-]\d{2}:\d{2}|Z)?\b/,
```

Comment:
Your change means that `... +0004` isn't accepted anymore.

```suggestion
		pattern: /\b\d{1,2}:\d{1,2}:\d{1,2}(?:[.,:]\d+)?(?:\s?[+-]\d{2}(?:\d{0,2}|:\d{2})|Z)?\b/,
```